faker/providers/address/el_GR/__init__.py

```diff
@@ -108,14 +108,17 @@ def city(self):
     def region(self):
         return self.random_element(self.regions)
 
+    # FIXME: latlng for el_GR should be moved to geo provider in default en_US locale
     def latlng(self):
         return float(self.latitude()), float(self.longitude())
 
+    # FIXME: latitude for el_GR overrides default latitude in geo provider without cause
```

Comment:
If these re-definitions of latitude and longitude were specific to the el_GR locale I would not have left FIXMEs, but they seem to be meaningless redefinitions overriding the global providers of the same name.

Reply:
I think the original idea was to return coordinates that are within Greece, which I think can be useful. The problem is that now you can't have coords out of Greece when using this locale.

 Maybe those methods should be called `local_latitude` and `local_longitude` or something like that.